mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/logging/impl/Log.java

```diff
@@ -403,4 +403,7 @@ SearchException cannotPurgeNonIndexedContainedType(
 			value = "The bind() method of binder '%1$s' is not implemented correctly:"
 					+ " it did not call context.setMarker(...).")
 	SearchException missingMarkerForBinder(Object binder);
+
+	@Message(id = ID_OFFSET_2 + 60, value = "The indexed embedded property '%1$s' of type '%2$s' does not provide any valid field.")
+	SearchException invalidIndexedEmbedded(String propertyName, String propertyType);
```

Comment:
Following the change in the implementation mentioned in my other comment, I'd rephrase the error like this:

```
	@Message(id = ID_OFFSET_2 + 60, value = "Type '%1$s' cannot be indexed-embedded, because no index mapping (@GenericField, @FullTextField, ...) is defined for that type.")
	SearchException invalidIndexedEmbedded(@FormatWith(PojoTypeModelFormatter.class) PojoTypeModel<?> typeModel); 
```

No need to mention the property name, it will be automatically included in the failure report (you can check that).